client/lib/domains/email-forwarding/has-email-forwards.js

```diff
@@ -0,0 +1,3 @@
+export function hasEmailForwards( domain ) {
+	return domain?.emailForwardsCount > 0;
```

Comment:
That's minor but we could just call the above function so only one of them actually cares about where to find that particular number of email forwards:

```suggestion
	return getEmailForwardsCount( domain ) > 0;
```

Reply:
yep, that makes sense